packages/graphql/src/translate/batch-create/parser.ts

```diff
@@ -35,15 +35,18 @@ function getRelationshipFields(
     const refNodes: Node[] = [];
 
     if (relationField) {
-        if (relationField.union) {
+        if (relationField.interface || relationField.union) {
+            throw new UnsupportedUnwindOptimization(`Not supported operation: Interface or Union`);
+        }
+     /*    if (relationField.union) {
             Object.keys(value as Record<string, any>).forEach((unionTypeName) => {
                 refNodes.push(context.nodes.find((x) => x.name === unionTypeName) as Node);
             });
         } else if (relationField.interface) {
             relationField.interface?.implementations?.forEach((implementationName) => {
                 refNodes.push(context.nodes.find((x) => x.name === implementationName) as Node);
             });
-        } else {
+        } */ else {
```

Comment:
What is the purpose of leaving this commented code here?

Reply:
It's the correct parser for the interfaces/unions with the unwind-create but because is not implemented yet and we raise an error before it will became an unreachable code with `relationField.interface/relationField.union` be of type `never`.
But I removed it and keep track of this in another branch!

Reply:
Sounds good!